layers/data_service_layer/python/data_service.py

```diff
@@ -24,4 +24,55 @@ def get_markers(self) -> List[dict]:
         :return: A list of markers.
         :raises NotImplementedError: Indicates the method is not yet implemented.
         """
-        raise NotImplementedError("This method is not implemented yet.")
+        try: 
+            response = self.table.scan()
+            markers = response.get('Items', [])
+            return markers
+        except Exception as e:
+            logger.error(f"Error retrieving markers")
+
+    def add_marker(self, markerId, name, longitude, latitude, imgurl, tags):
```

Comment:
@PCBush I suggest to use the implemented `LocationMarker` class as a parameter here (`from location_marker import LocationMarker` at the top). Like this
`def add_marker(self, locationMarker: LocationMarker):`